### ISSUE TITLE
Cannot install atom on Ubuntu 16.10

### ISSUE THREAD

**Issue by CasperHK** [atom/atom#12971]
### Prerequisites

* [x] Can you reproduce the problem in [safe mode](http://flight-manual.atom.io/hacking-atom/sections/debugging/#using-safe-mode)?
* [x] Are you running the [latest version of Atom](http://flight-manual.atom.io/hacking-atom/sections/debugging/#update-to-the-latest-version)?
* [ ] Did you check the [debugging guide](http://flight-manual.atom.io/hacking-atom/sections/debugging/)?
* [ ] Did you check the [FAQs on Discuss](https://discuss.atom.io/c/faq)?
* [ ] Are you reporting to the [correct repository](https://github.com/atom/atom/blob/master/CONTRIBUTING.md#atom-and-packages)?
* [ ] Did you [perform a cursory search](https://github.com/issues?q=is%3Aissue+user%3Aatom+-repo%3Aatom%2Felectron) to see if your bug or enhancement is already reported?

For more information on how to write a good [bug report](https://github.com/atom/atom/blob/master/CONTRIBUTING.md#how-do-i-submit-a-good-bug-report) or [enhancement request](https://github.com/atom/atom/blob/master/CONTRIBUTING.md#how-do-i-submit-a-good-enhancement-suggestion), see the `CONTRIBUTING` guide.

### Description

I downloaded the .deb file from the official website but it cannot be used to install atom on ubuntu 16.10 in the GUI mode. The `install` button does not prompt me to input the root password of ubuntu system. Then, I tried to use command `sudo dpkg -i '/home/casper/Downloads/atom-amd64.deb'`and I got the following:

```
casper@casper-Z97X-UD3H:~$ sudo dpkg -i '/home/casper/Downloads/atom-amd64.deb' [sudo] password for casper: 
(Reading database ... 176587 files and directories currently installed.)
Preparing to unpack .../Downloads/atom-amd64.deb ...
Unpacking atom (1.11.1) over (1.11.1) ...
dpkg: dependency problems prevent configuration of atom:
 atom depends on git; however:
  Package git is not installed.

dpkg: error processing package atom (--install):
 dependency problems - leaving unconfigured
Processing triggers for bamfdaemon (0.5.3+16.10.20160929-0ubuntu1) ...
Rebuilding /usr/share/applications/bamf-2.index...
Processing triggers for gnome-menus (3.13.3-6ubuntu4) ...
Processing triggers for desktop-file-utils (0.23-1ubuntu1) ...
Processing triggers for mime-support (3.60ubuntu1) ...
Errors were encountered while processing:
 atom
casper@casper-Z97X-UD3H:~$ 

```

**Expected behavior:** 

Install atom normally on Ubuntu.

**Actual behavior:** 

Cannot install atom on Ubuntu.

**Comment by Ben3eeE** [comment]
>dpkg: dependency problems prevent configuration of atom:
 atom depends on git; however:
  Package git is not installed.

Install git before installing Atom.

**Comment by eogidan** [comment]
I have git already installed and up to date, and I still have the problem

**Comment by harrypotter0** [comment]
Hi guys I am facing some problems with unmet dependencies . 
What I tried : 
`sudo dpkg -i atom-amd64.deb`

I am facing these errors : 
`(Reading database ... 474533 files and directories currently installed.)
Preparing to unpack atom-amd64.deb ...
Unpacking atom (1.21.1) over (1.21.1) ...
dpkg: dependency problems prevent configuration of atom:
 atom depends on python; however:
  Package python is not installed.

dpkg: error processing package atom (--install):
 dependency problems - leaving unconfigured
Processing triggers for desktop-file-utils (0.22-1ubuntu5.1) ...
Processing triggers for gnome-menus (3.13.3-6ubuntu3.1) ...
Processing triggers for bamfdaemon (0.5.3~bzr0+16.04.20160824-0ubuntu1) ...
Rebuilding /usr/share/applications/bamf-2.index...
Processing triggers for mime-support (3.59ubuntu1) ...
Errors were encountered while processing:
 atom`
Please help me resolve this .

**Comment by Ben3eeE** [comment]
@harrypotter0 Please see https://flight-manual.atom.io/getting-started/sections/installing-atom/#platform-linux for an installation guide.

For most questions about Atom there are a lot of options, check out:

* [The Atom Flight Manual](http://flight-manual.atom.io)
* [The Atom FAQ on Discuss](https://discuss.atom.io/c/faq)
* [Discuss, the official Atom and Electron message board](https://discuss.atom.io)
* [Join the Atom Slack team](http://atom-slack.herokuapp.com/)

On Discuss and in the Atom Slack team, there are a bunch of helpful community members that should be willing to point you in the right direction.

**Comment by harrypotter0** [comment]
@Ben3eeE  I am facing some python issues (basically atom dependency on Python ) I was following the same manual for it's installation but some unmet dependency has erupted , help me in resolving them :)

**Comment by harrypotter0** [comment]
Earlier I used this thread https://github.com/atom/atom/issues/14358 but couldn't resolve issues .
Now it's solved all unmet dependencies are met.
Here are the commands I used : 

1. Removed atom completely using `sudo apt-get remove atom`

2. Checked for the dependencies again  `sudo dpkg --configure -a` .

3. Removed python-minimal which was producing problem using `sudo dpkg -P python-minimaltall python-minimal `.

4.Then checked for the deendencies problem again using `sudo dpkg --configure -a` and it's all clear didn't produced any errors . 
Thanks for your effort @Ben3eeE  .
